content/providers/hardware.json

```diff
@@ -0,0 +1,162 @@
+[
+  {
+    "title": "IBM Qiskit Runtime",
+    "image": "static/images/providers/qiskit-ibm-runtime.png",
+    "description": "Qiskit Runtime is a quantum computing service and programming model that allows users to optimize workloads and efficiently execute them on quantum systems at scale. The programming model extends the existing interface in Qiskit with a set of new primitive programs.",
+    "installation": "pip install qiskit-ibm-runtime",
+    "websiteCta": {
+      "label": "Website",
+      "url": "https://quantum-computing.ibm.com/lab/docs/iql/runtime/"
+    },
+    "docCta": {
+      "label": "Docs",
+      "url": "https://qiskit.org/documentation/partners/qiskit_ibm_runtime/"
+    },
+    "sourceCta": {
+      "label": "GitHub",
+      "url": "https://github.com/Qiskit/qiskit-ibm-runtime"
+    },
+    "helloWorldExample": [
+      "from qiskit import QuantumCircuit\n",
+      "from qiskit_ibm_runtime import QiskitRuntimeService, Sampler\n",
+      "\n",
+      "# Save an IBM Quantum account\n",
+      "QiskitRuntimeService.save_account(channel=\"ibm_quantum\", token=\"MY_IBM_QUANTUM_TOKEN\")\n",
+      "\n",
```

Comment:
It looks like you're adding this line to separate blocks. Could you use `&nbsp;` instead of `\n`? (Until we find a better solution)

Reply:
Fixed in https://github.com/Qiskit/qiskit.org/pull/2968/commits/60fb2c5174ff6e20fb77418e13bb75ca4fa0ea89